source/common/quic/platform/quiche_mutex_impl.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
```

Comment:
This file looks identical to https://github.com/google/quiche/blob/main/common/platform/default/quiche_platform_impl/quiche_mutex_impl.h with method definitions in https://github.com/google/quiche/blob/main/common/platform/default/quiche_platform_impl/quiche_mutex_impl.cc inlined.  Could it be removed just like quic_testvalue_impl.h was removed?

Reply:
AH! Yes, good point. Done.